src/node/handler/PadMessageHandler.js

```diff
@@ -1004,7 +1004,14 @@ function handleClientReady(client, message)
   var historicalAuthorData = {};
   var currentTime;
   var padIds;
-
+  var loginUsr = null;
+  
+  if(settings.requireAuthentication){
+    var authStr = client.conn.request.headers.authorization;
+    loginUsr = new Buffer(authStr.split(' ')[1], 'base64').toString('ascii').split(':')[0];
+  }
```

Comment:
You are assuming `authorization` header has an specific format -- and more than that: it is not `undefined`. This is not always true: I can set `requireAuthentication: true` on `settings.json` and still get no `client.conn.request.headers.authorization`.

Please provide a safer way to get `loginUsr` (one that won't throw an exception, I mean).